Dockerfile

```diff
@@ -1,3 +1,10 @@
+FROM alpine:latest as alpine
+RUN apk --update add unzip && rm -rf /var/cache/apk/*
```

Comment:
Could be reduced in:
```
RUN apk --no-cache add unzip
```

`--update` is unnecessary because you are already targeting the latest alpine linux version.